qsimcirq/_version.py

```diff
@@ -1,3 +1,3 @@
 """The version number defined here is read automatically in setup.py."""
 
-__version__ = "0.21.0"
+__version__ = "0.22.0.dev0"
```

Comment:
Is it supposed to be 0.22.0.dev0 or 0.22.0.dev?

Reply:
The [PyPA guide to valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#valid-version-numbers) uses `x.y.z.devN`, and this is what Qualtran and Cirq are doing. See, for example, [Qualtran's version file](https://github.com/quantumlib/Qualtran/blob/main/qualtran/_version.py).

Reply:
(BTW, I know the CI checks are failing, and I'm also working on getting them running.)